CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.2
```

Comment:
✨ Nice to start a changelog from the start of a project.

Reply:
`pub.dev` strongly encourages and yells at me when publish without a corresponding change